prombench/manifests/prombench/benchmark/3b_prometheus-test-pr_deployment.yaml

```
@@ -93,6 +104,12 @@ spec:
           path: /mnt/disks/ssd0
       - name: prometheus-executable
         emptyDir: {}
+      - name: bucket-config   # Define the Secret volume
+        secret:
+          secretName: bucket-secret
+      - name: key
```

```suggestion
      - name: blocksync-location-key
```
The above is just an example.
Let's be more specific. `key` is too generic.